specification/cognitiveservices/OpenAI.Inference/tspconfig.yaml

```diff
@@ -29,6 +29,8 @@ options:
     generate-tests: false
     generate-samples: false
     customization-class: customization/src/main/java/OpenAICustomizations.java
+    custom-types-subpackage: "implementation.models"
+    custom-types: "UploadFileRequest"
```

Comment:
The model is anonymous in TypeSpec source. Hence we had to use this to ref it and modify its package..